test/e2e/e2e_suite_test.go

```diff
@@ -25,6 +25,7 @@ import (
 
 	"github.com/openshift-kni/mixed-cpu-node-plugin/test/e2e/fixture"
 	"github.com/openshift-kni/mixed-cpu-node-plugin/test/e2e/infrastructure"
+	_ "github.com/openshift-kni/mixed-cpu-node-plugin/test/e2e/mixedcpus"
```

Comment:
this seems correct though - did you notice the tests weren't run before or after adding this change?

Reply:
Before. Now it's fine